deployment/ansible/group_vars/production.example

```diff
@@ -0,0 +1,241 @@
+# languages. The id: field should generally be the ISO 639-1 language code for the language in
+# question, with an optional lowercase ISO 3166-1 alpha-2 country code to specify the regional
+# variant. However, in all cases, the id: field needs to exactly match the filename of the
+# corresponding file in web/app/i18n, if the file extension (.json) is removed.
+# Set the rtl: key to true for right-to-left languages.
+languages:
+    - { id: "en-us", label: "English", rtl: false }
+
+# DRIVER uses the DarkSky (formerly forecast.io) API to obtain weather data, which requires an API
+# key. To obtain an API key, see here: https://darksky.net/dev
+forecast_io_api_key: ""
+
+## OIDC SETTINGS
+# DRIVER has the capability to enable Single-Sign-On via OAuth (usually via Google).
+# To enable this feature, you will need a client ID and secret, which can be obtained from the
+# Google API Console: https://developers.google.com/identity/sign-in/web/devconsole-project
+# The URLs that you will need to authorize for your application are:
+# https://<your domain name>
+# https://<your domain name>/
+# https://<your domain name>/openid/
+# https://<your domain name>/openid/callback/
+# https://<your domain name>/openid/callback/login
+# https://<your domain name>/openid/callback/login/
+# https://<your domain name>/openid/callback/logout
+# https://<your domain name>/openid/callback/logout/
+oauth_client_id: ""
+oauth_client_secret: ""
+
+## Localization settings
+# The name of the time zone in which you want your DRIVER instance to operate; this should usually
+# be the time zone for your country. If your country has multiple time zones, pick one. A list of
+# valid time zone names can be found here:
+# https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+local_time_zone_id: "Asia/Manila"
+
+# The ISO 3166-1 alpha 2 country code for the country where this DRIVER instance will be operated.
+# https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2
+# Note that this may be different from the language code used above.
+local_country_code: "ph"
+# The latitude and longitude of where the maps in your installation of DRIVER should center by
+# default. This is given as [lat, lon]
+local_center_lat_lon: [14.5995, 120.984]
+# DRIVER has the ability to calculate "black spots", which are stretches of road or intersections
+# that experience relatively higher crash rates. The calculation process uses extracts of Open
+# Street Map data. This parameter specifies where to download that extract from. GeoFabrik maintains
+# extracts for most countries and regions in the world; for most countries, you should be able to
+# simply search https://download.geofabrik.de until you find your country and then use the link for
+# your country's extract. If you are trying to run DRIVER on a sub-national area, you may want to
+# generate your own extract for efficiency (the black spot calculation will run more quickly on
```

Comment:
Docs on how to generate their own extract? Like "download the `.osm` and convert to a protobuf for your area of interest"?

Reply:
Thinking more on this, I don't think that asking someone to generate _and host_ their own OSM extracts is very reasonable. Luckily, the OSM wiki has links to a bunch more extract resources that have cities and sub-national extracts, so I'm going to update to include a link to that.